diff --git a/plugins/GeoIp2/LocationProvider/GeoIp2/Php.php b/plugins/GeoIp2/LocationProvider/GeoIp2/Php.php
index 10ce942350..1f80a2fb11 100644
--- a/plugins/GeoIp2/LocationProvider/GeoIp2/Php.php
+++ b/plugins/GeoIp2/LocationProvider/GeoIp2/Php.php
@@ -319,8 +319,8 @@ class Php extends GeoIp2
             $str1 = transliterator_transliterate('Any-Latin; Latin-ASCII', $str1);
             $str2 = transliterator_transliterate('Any-Latin; Latin-ASCII', $str2);
         } elseif (function_exists('iconv')) {
-            $str1 = iconv('UTF-8', 'ASCII//TRANSLIT', $str1);
-            $str2 = iconv('UTF-8', 'ASCII//TRANSLIT', $str2);
+            $str1 = @iconv('UTF-8', 'ASCII//TRANSLIT', $str1) ?: $str1;
+            $str2 = @iconv('UTF-8', 'ASCII//TRANSLIT', $str2) ?: $str2;
         }
 
         return strtolower($str1) === strtolower($str2);
