diff --git a/plugins/CoreHome/CoreHome.php b/plugins/CoreHome/CoreHome.php
index 75425a55f8..4e6e069d55 100644
--- a/plugins/CoreHome/CoreHome.php
+++ b/plugins/CoreHome/CoreHome.php
@@ -136,6 +136,7 @@ class CoreHome extends \Piwik\Plugin
         $stylesheets[] = "plugins/Morpheus/stylesheets/base/icons.css";
         $stylesheets[] = "plugins/Morpheus/stylesheets/base.less";
         $stylesheets[] = "plugins/Morpheus/stylesheets/main.less";
+        $stylesheets[] = "plugins/CoreHome/stylesheets/a11y.less";
         $stylesheets[] = "plugins/CoreHome/stylesheets/coreHome.less";
         $stylesheets[] = "plugins/CoreHome/stylesheets/dataTable.less";
         $stylesheets[] = "plugins/CoreHome/stylesheets/cloud.less";
@@ -205,6 +206,7 @@ class CoreHome extends \Piwik\Plugin
 
     public function getClientSideTranslationKeys(&$translationKeys)
     {
+        $translationKeys[] = 'CoreHome_ReportConfiguration';
         $translationKeys[] = 'General_Export';
         $translationKeys[] = 'General_InvalidDateRange';
         $translationKeys[] = 'General_Loading';
diff --git a/plugins/CoreHome/lang/en.json b/plugins/CoreHome/lang/en.json
index 0a050374c1..c1c9bc454a 100644
--- a/plugins/CoreHome/lang/en.json
+++ b/plugins/CoreHome/lang/en.json
@@ -1,5 +1,6 @@
 {
     "CoreHome": {
+        "ReportConfiguration": "Report Configuration",
         "AIInsights": "AI Insights",
         "CategoryNoData": "No data in this category. Try to \"Include all population\".",
         "ChangeVisualization": "Change visualization",
diff --git a/plugins/CoreHome/stylesheets/a11y.less b/plugins/CoreHome/stylesheets/a11y.less
new file mode 100644
index 0000000000..cb48a85973
--- /dev/null
+++ b/plugins/CoreHome/stylesheets/a11y.less
@@ -0,0 +1,9 @@
+.visually-hidden {
+    clip: rect(0 0 0 0);
+    clip-path: inset(50%);
+    height: 1px;
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+}
diff --git a/plugins/CoreHome/vue/src/DataTable/DataTableActions.vue b/plugins/CoreHome/vue/src/DataTable/DataTableActions.vue
index 08c74000f4..3e4981a01c 100644
--- a/plugins/CoreHome/vue/src/DataTable/DataTableActions.vue
+++ b/plugins/CoreHome/vue/src/DataTable/DataTableActions.vue
@@ -19,6 +19,7 @@
       v-if="hasConfigItems && (isAnyConfigureIconHighlighted || isTableView)"
     >
       <span class="icon-configure"></span>
+      <span class="visually-hidden">{{ translate('CoreHome_ReportConfiguration') }}</span>
     </a>
 
     <a v-if="hasFooterIconsToShow"
@@ -94,7 +95,10 @@
       href=""
       style="margin-right:3.5px"
       @click.prevent
-    ><span class="icon-export"></span></a>
+    >
+      <span class="icon-export"></span>
+      <span class="visually-hidden">{{ translate('General_ExportThisReport') }}</span>
+    </a>
 
     <a
       v-if="showExportAsImageIcon"
